app/Http/Controllers/UserController.php

```diff
@@ -69,12 +86,14 @@ public function edit(User $user)
      * @param  \App\Models\User  $user
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, User $user)
+    public function update(UserUpdateRequest $request, User $user)
     {
-        $user->username = $request->name;
+        $validated = $request->validated();
+
+        $user->username = $validated['name'];
         $user->save();
 
-        return redirect()->back();
+        return redirect('/users');
```

Comment:
ko dùng URL fix cứng.
```suggestion
        return redirect()->route('users.index');
```